dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/LogConfigurationTask.java

```diff
@@ -34,6 +38,7 @@
 public class LogConfigurationTask extends Task {
 
     private final ILoggerFactory loggerContext;
+    private final Timer timer = new Timer(LogConfigurationTask.class.getSimpleName(), true);
```

Comment:
Thinking about wrapping this in a lazy init block to delay creating the timer until the first expireable log level is submitted.